app/views/state_file/questions/md_had_health_insurance/edit.html.erb

```diff
@@ -0,0 +1,91 @@
+<% filing_year = Rails.configuration.statefile_current_tax_year %>
```

Comment:
```suggestion
```
doesn't look like you use this anywhere? also for future reference there's a helper method in `StateFile::StateFileControllerConcern`, available here because it's included on `StateFile::Questions::QuestionsController`, called `current_tax_year` which gets this out of the config